docs/book/products/multi_source_inventory.rst

```diff
@@ -73,6 +73,11 @@ It's possible to add more inventory sources filter, with higher or lower priorit
 
    How to create a custom Inventory Sources Filter? Read :doc:`this Cookbook </cookbook/index>`.
 
+Resolving InventorySourceStock for ordered products
+---------------------------------------------------
+
+Before in Sylius, inventory was resolved after payment, in Sylius Plus we changed it and now after payment all items are reserved and after shipment inventoryStock is reduced
```

Comment:
IMHO this feature is more about orders fulfillment/shipments rather than `products/multi_source_inventory`.
```suggestion
.. warning::
Standard Sylius distribution is releasing a stock inventory when the whole order is paid, while in Plus version, it has been switched to be released after shipment has been shipped.
```

Also, IIRC, the stock is still reserved when the order is placed, not `after payment`

Reply:
i spoke with @CoderMaggie , and inventory is a better place for this information :)